src/main/java/broski/output/Ui.java

```diff
@@ -0,0 +1,227 @@
+package broski.output;
+
+import java.util.ArrayList;
+
+import broski.exception.DeadlineException;
+import broski.exception.EventException;
+import broski.exception.TodoException;
+import broski.exception.WrongInputException;
+import broski.parser.DateTimeParser;
+import broski.parser.Parser;
+import broski.task.Deadline;
+import broski.task.Event;
+import broski.task.Task;
+import broski.task.Todo;
+import broski.taskRelated.TaskList;
+
+/**
+ * Class that handles output such as println.
+ */
+public class Ui {
+    private static final String LINE = "_________________________________________";
+
+    private static void printLine() {
+        System.out.println(LINE);
+    }
+
+    /**
+     * Greets the user.
+     */
+    public void greeting() {
+        printLine();
+        System.out.println("Wassup! I'm Broski!");
+        System.out.println("What can I do for you bro?");
+        printLine();
+    }
+
+    /**
+     * Says bye to the user.
+     */
+    public void exit() {
+        printLine();
+        System.out.println("Bye, bro. See ya around!");
+        printLine();
+    }
+
+    /**
+     * Displays the entire list to the user.
+     * @param taskList stores all prior task input.
+     */
+    public void list(TaskList taskList) {
+        printLine();
+        for (int i = 1; i <= taskList.size(); i++) {
+            System.out.println(i + ". " + taskList.get(i - 1));
+        }
+        printLine();
+    }
+
+    /**
+     * Informs the user that the task has been marked as done.
+     * @param taskList stores all prior task input.
+     * @param parser parse user input.
+     * @param reply user input.
+     */
+    public void mark(TaskList taskList, Parser parser, String reply) {
+        printLine();
+        int i = parser.parseIndex(reply);
+        taskList.get(i).mark();
+        System.out.println("Solid! Marked as done for you:");
+        System.out.println(taskList.get(i));
+        printLine();
+    }
+
+    /**
+     * Informs the user that the task has been marked as incomplete.
+     * @param taskList stores all prior task input.
+     * @param parser parse user input.
+     * @param reply user input.
+     */
+    public void unmark(TaskList taskList, Parser parser, String reply) {
+        printLine();
+        int i = parser.parseIndex(reply);
+        taskList.get(i).unmark();
+        System.out.println("Alright, I've marked the task as undone:");
+        System.out.println(taskList.get(i));
+        printLine();
+    }
+
+    /**
+     * Informs the user that the task has been deleted.
+     * @param taskList stores all prior task input.
+     * @param parser parse user input.
+     * @param reply user input.
+     */
```

Comment:
I like that your Javadoc comments are very informative and have the correct form!